addons/web/static/src/js/core/error_utils.js

```diff
@@ -46,6 +46,7 @@ export async function annotateTraceback(error) {
     const subst = `:$1`;
     error.stack = error.stack.replace(regex, subst);
   }
+  // eslint-disable-next-line no-undef
   const frames = await StackTrace.fromError(error);
```

Comment:
Probably makes more sense to declare it as a global, as provided by stracktracejs?

Reply:
stacktrace.js is loaded on line 40, and it's the only place on Odoo that it's used, is for those reasons that we don't declare it as global.